.github/workflows/main.yml

```diff
@@ -43,6 +43,9 @@ jobs:
       run: |
             sudo apt-get update -q -y
             sudo apt-get install -q -y libsdl2-dev libsdl2-mixer-dev
+            wget https://apt.llvm.org/llvm.sh
+            sudo chmod +x ./llvm.sh
+            sudo ./llvm.sh 17
```

Comment:
Inside the guest environment initiated by [run-on-arch-action](https://github.com/uraimo/run-on-arch-action), the command `sudo` is not allowed. Instead, superuser (aka "root") is the only user there.